RELEASE_NOTES.md

```diff
@@ -36,14 +38,13 @@ This is built on the Jupyter Notebook v6.4.12 and IPython 8.5.0 (more notes will
   - @wdio/browserstack-service: 8.12.2 -> 8.29.3
   - @wdio/cli: 8.12.2 -> 8.29.3
   - axios: 1.4.0 -> 1.6.1
-  - chromedriver: 114.0.2 -> 121.0.0
+  - chromedriver: 114.0.2 -> 123.0.0
```

Comment:
Mostly, this was put in to test out whether the fails were because of a headless chrome incompatibility. And release notes weren't updated in the previous PR.